apps/trading/components/vega-wallet-connect-button/vega-wallet-connect-button.tsx

```diff
@@ -86,7 +89,7 @@ export const VegaWalletConnectButton = ({
         >
           <div className="min-w-[340px]" data-testid="keypair-list">
             <TradingDropdownRadioGroup
-              value={pubKey}
+              value={pubKey || undefined}
```

Comment:
can pubkey be null here?

Reply:
No pubKey can never be null. The default state for an unconnected wallet is to have pubKey as undefined. 

This made it much easier to work with a persisted zustand store, when I needed to partialize what was put into local storage